src/pages/home/wallet/UnclaimedRoyalties.tsx

```diff
@@ -0,0 +1,63 @@
+import { Box, Button } from "@mui/material";
+import { Typography } from "elements";
+import theme from "theme";
+
+interface UnclaimedRoyaltiesProps {
+  unclaimedRoyalties: number;
+}
+
+export const UnclaimedRoyalties = ({
+  unclaimedRoyalties,
+}: UnclaimedRoyaltiesProps) => {
+  return (
+    <Box
+      sx={ {
+        backgroundColor: theme.colors.grey600,
+        padding: 2.5,
+        maxWidth: "400px",
+        height: "100px",
+        borderRadius: "8px",
+        display: "flex",
+        justifyContent: "space-between",
+      } }
+    >
+      <Box
+        sx={ {
+          display: "flex",
+          flexDirection: "column",
+          justifyContent: "space-between",
+          paddingRight: [1, "unset"],
+        } }
+      >
+        <Typography color="grey100" fontSize={ 12 }>
+          UNCLAIMED{ " " }
+          <Box
+            component="span"
+            sx={ {
+              display: ["none", "inline"],
+            } }
+          >
+            ROYALTIES
+          </Box>
+        </Typography>
+        <Typography fontSize="28px" fontWeight={ 700 }>
+          ${ unclaimedRoyalties }
+        </Typography>
+      </Box>
+      <Button
```

Comment:
*Feedback requested comment*

Should we be using one of our button variants instead or maybe add it as an option if we plan to reuse this button style?

Reply:
Good call. Created a new component in elements.